app/views/papers/_index.html.erb

```diff
@@ -32,8 +32,8 @@
       <small>
         <%= paper.authors.map(&:name).join(', ') %>
         <i>
-          in Journal of Reserved
-          (<%= paper.published_at %>)
+          <%= "in " + paper.publication.name.split.map(&:capitalize).join(' ') unless paper.publication.nil? %>
+          <%= "(" + paper.published_at.to_s + ")" unless paper.published_at.nil? %>
```

Comment:
yea, sometimes pubmed doesn't have it, so unless we add a default, they will be nil.